src/config.rs

```diff
@@ -299,6 +473,30 @@ impl HookBuilder {
         }
     }
 
+    // XXX the following tip is a bit hacky, but it's an easy way to test new styles
+
+    /// Set the global styles that `color_eyre` should use.
+    /// 
+    /// Tip: An easy way to test new styles is the following:
+    ///
+    /// 1) clone `color_eyre` via `git clone https://github.com/yaahc/color-eyre.git`
+    /// 
+    /// 2) `cd color-eyre`
+    ///
+    /// 3) open "tests/styles.rs"
+    ///
+    /// 4) search for "# Easy way to test styles"
+    ///
+    /// 5) follow the instructions there
+    ///
+    /// This, basically, leverages the test code to display an error message that includes
+    /// all elements that can be styled. You then can define your own styles which will
+    /// be displayed.
```

Comment:
I think the recommendation is good but I would tweek it slightly for brevity. I'd want to make it as an example rather than a test and just tell them to check out the test if they want to play with their own style, we already do this a lot with other features that we document in the `README`.

```suggestion
    /// Set the global styles that `color_eyre` should use.
    /// 
    /// **Tip:** You can test new styles by editing `examples/styles.rs` in the `color-eyre` repository.
```